### ISSUE TITLE
Implement version of the library without parser

### ISSUE THREAD

**Issue by agershun** [agershun/alasql#219]
Instead SQL text code, this function will accept AST as an input, like:
```js
    var res = alasql.run({statement:'=', expression;{value:10}});
```
This can save about 100 kb and allow to implement other interfaces for other JS languages, e.g. Clojure

**Comment by mathiasrw** [comment]
Is the idea to change the AST with this syntax internally? Otherwise it could be another "flavour" for querying

**Comment by agershun** [comment]
Yes. The idea to create something like LINQ and ORM and MongoDB plus external JSON interface
